infrastructure/conf/nginx-conf-local/default

```diff
@@ -23,28 +20,15 @@ server {
         log_not_found off;
     }
 
+    location = /robots.txt {
+        alias /home/site/wwwroot/infrastructure/conf/dev.robot.txt;
+    }
+
     # Permanent redirects to avoid dead links
     location = /en/communities/digital-talent { return 301 https://www.canada.ca/en/government/system/digital-government/gcdigital-community.html; }
     location = /fr/communities/digital-talent { return 301 https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/collectivite-gcnumerique.html; }
 
-    # local auth
```

Comment:
umm.. Idk how come these removed.. gonna put it back

Reply:
Yeah, if you run the post_deployment script inside the docker container it will copy the deploy version of the script to the nginx conf location - which is where the local version of the script is mounted.  :disappointed:  It's a gotcha but I couldn't think of a solution and I figured people wouldn't be running that deployment script locally very often.

Reply:
aah .. ok . It make sense now. Thanks! Rolled it back to the prev version & added my change only.